index.js

```diff
@@ -94,6 +94,41 @@ module.exports.init = function (func) {
       }
     };
 
+    var arpeggiate = function (obj, start, duration, callback) {
+      var notes;
+
+      start = ctx.currentTime + (start || 0);
+      duration = duration || defaultDuration;
+
+      if (callback) callback = _.partial(callback, obj);
+
+      // For a chord, play every note in succession
+      if (s11.chord.isChord(obj)) {
+        notes = ensureOctave(obj).chord;
+        _.each(notes, function (note, i) {
+          playNote(note, start + i * duration, duration, callback);
+        });
+      }
+      // For a scale, play notes in succession
+      else if (s11.scale.isScale(obj)) {
+        notes = ensureOctave(obj).scale.concat(obj.root.transpose('P8'));
+        _.each(notes, function (note, i) {
+          playNote(note, start + i * duration, duration, callback);
+        });
+      }
+      // For an array of notes, play notes in succession
+      else if (obj instanceof Array) {
+        notes = _.map(obj, ensureOctave);
+        _.each(notes, function (note, i) {
+          playNote(note, start + i * duration, duration, callback);
+        });
+      }
+      // Otherwise, assume we have a note and play it
```

Comment:
Since `arpeggiate` is a specific thing whereas `play` is meant to be a catch-all, I think the else case should either throw or do nothing.